src/named-node.ts

```diff
@@ -1,55 +1,58 @@
 'use strict'
 import ClassOrder from './class-order'
 import Node from './node-internal'
+import { NamedNodeTermType, TermType} from './types'
+import { termValue } from './utils/termValue'
+import { TFNamedNode } from './tf-types'
 
 /**
- * @class NamedNode
- * @extends Node
+ * A named (IRI) RDF node
  */
-export default class NamedNode extends Node {
+export default class NamedNode extends Node implements TFNamedNode {
+  static termType = TermType.NamedNode
+
+  classOrder = ClassOrder.NamedNode
+  termType: NamedNodeTermType = TermType.NamedNode
```

Comment:
If I leave the non-static one out, Typescript says:

```
'import("/Users/joep/dev/src/github.com/linkeddata/rdflib.js/src/named-node").default' incorrectly implements interface 'import("/Users/joep/dev/src/github.com/linkeddata/rdflib.js/src/tf-types").NamedNode'.
  Types of property 'termType' are incompatible.
    Type 'TermType' is not assignable to type '"NamedNode"'.
      Type '"BlankNode"' is not assignable to type '"NamedNode"'.ts(2420)
```

Can't understand why, though...

Reply:
That's because `TFNamedNode` defines a non-static member `termType`. Is the `static` version used somewhere? A search for `NamedNode.termType` doesn't turn up any results for me...

Reply:
The class (static) method isn't used, you're right. Removed the static properties.